corehq/apps/app_manager/models.py

```diff
@@ -3899,6 +3914,7 @@ def validate_app(self):
                 if xmlns_count[xmlns] > 1:
                     errors.append({'type': "duplicate xmlns", "xmlns": xmlns})
 
+        self.ensure_module_unique_ids(should_save=True)
```

Comment:
This is check before build - if there are apps that don't have module.unique_id. Added this check while importing already, but adding here as well, so that priori imported apps work won't give this error.